### PR TITLE
JC-2244 Remove column STATE from POST table

### PR DESCRIPTION
**Pull request by dsdolzhenko**
Pull request for issue [JC-2244](http://jira.jtalks.org/browse/JC-2244)

### CODE REVIEW

File: jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+alter table POST drop column STATE;
```
Comment (ctapobep) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql]: Could you please also leave a comment for which task this is done and maybe why.

Comment (dsdolzhenko) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql]: done

Comment (ctapobep) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql]: Ok, thanks, could you squash these 2 commits into 1?

Comment (dsdolzhenko) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql]: done

**Pull request merged**